egs/mgb2/asr/local/mgb_format_data.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
```

Comment:
I think we don't need this script.
Please remove it.

Reply:
done